plugins/lang-plan-a/src/main/java/org/elasticsearch/plan/a/PlanAParserBaseVisitor.java

```diff
@@ -0,0 +1,357 @@
+// ANTLR GENERATED CODE: DO NOT EDIT
+package org.elasticsearch.plan.a;
```

Comment:
I don't think laziness should be disguised as code review comments (forcing someone to answer your questions / be your LMGTFY.com).

If you haven't read the code, then its not a review.